angr/procedures/win32/GetLastInputInfo.py

```diff
@@ -9,11 +9,11 @@
 #   PLASTINPUTINFO plii
 # );
 
-# typedef struct tagLASTINPUTINFO {
-#   UINT  cbSize;  // The size of the structure, in bytes.
-#                  // This member must be set to sizeof(LASTINPUTINFO).
-#   DWORD dwTime;
-# } LASTINPUTINFO, *PLASTINPUTINFO;
+#| typedef struct tagLASTINPUTINFO {
```

Comment:
This file probably shouldn't be modified in this PR?

Reply:
One of the `ruff` things aggressively checks for python 3.4 type comments, which are comments starting with `# type`; hence I changed this to `#| type` to avoid the false positive. I hopefully fixed all our type comments in other PRs.

Reply:
This is autogenerated though, no? So shouldn't we fix that first instead of fixing it here?

Reply:
Oh, win32 procedures are sometimes... Hmm, I'm not sure how this is made then. We could just disable `ruff` for the `win32` directory and other auto-generated files? (Though then again, it did catch bugs in them already...). @rhelmot ?

Reply:
That file is not autogenerated. None of the simprocedure class definitions are autogenerated, just the prototype declarations.